pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -109,7 +109,7 @@ var (
 	// isMultiVCSupportEnabled is set to true only when the MultiVCenterCSITopology FSS
 	// is enabled. isMultivCenterCluster is set to true only when the MultiVCenterCSITopology FSS
 	// is enabled and the K8s cluster involves multiple VCs.
-	isMultiVCSupportEnabled, isMultivCenterCluster bool
```

Comment:
why are we removing isMultivCenterCluster. When fss is enabled, it does not mean we have multi vCenter deployment. Can you make sure it does not break anything.

Reply:
The code which used this value is no longer required. That is the reason I removed the variable. It is local to this package